### ISSUE TITLE
In DIRECT_MENTION, tagged bot name also comes as part of the message

### ISSUE THREAD

**Issue by abstract-karshit** [rampatra/jbot#52]
HI,
When I add bot in a group and than tag it with a message(DIRECT_MENTION), doing event.getText() gives me message along with some <--id--> and simply text.
Same thing in howdy botkit gives me only message and no tagging information if I don message .text there.
Any help?

**Comment by ramswaroop** [comment]
Yes, I think I am sending the entire message including the mentions. You can replace `@whatevertext` with `` empty.
